client/src/components/History/HistoryViews/JobHistoryModal.jsx

```diff
@@ -236,7 +244,7 @@ function JobHistoryModal({ jobId }) {
             <JobHistoryContent
               jobName={jobName}
               jobDetails={job}
-              jobDisplay={display}
+              jobDisplay={display ? display : {}}
```

Comment:
Instead of this, we could set `jobDisplay` with a default prop of `{}` in the proptypes for `JobHistoryContent`